me.sergiotarxz.Exd.yml

```diff
@@ -0,0 +1,120 @@
+command: hiperthermia
+modules: 
+  - name: 'imagemagick'
+    sources:
+      - type: 'git'
+        url: 'https://github.com/ImageMagick/ImageMagick'
+        commit: '5fb1f0f0fca16dabf604fdd07c9c935213b0074d'
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --disable-docs
+      - --with-modules
+      - --with-threads
+  - name: 'gd'
+    sources:
+      - type: "git"
+        url: 'https://github.com/libgd/libgd'
+        commit: 'c2867fd56dd3198ca4d09572913166d60fb657f9'
+  - name: "perl"
+    no-autogen: true
+    config-opts:
+        - '-des'
+        # Build a shared library.
+        - '-Duseshrplib'
+    build-options:
+        cflags: '-fPIC'
+        ldflags: '-fpic'
+    sources:
+        - type: archive
+          url: https://www.cpan.org/src/5.0/perl-5.40.0.tar.gz
+          sha256: c740348f357396327a9795d3e8323bafd0fe8a5c7835fc1cbaba0cc8dfe7161f
+        - type: shell
+          commands:
+            # Have Flatpak run the GNU-compatible configure script.
+            - 'ln -s configure{.gnu,}'
+      # Restore write permission to the Perl libraries.
+    post-install:
+        - 'chmod -R u+w /app/lib/perl5'
+      # Clean up a bunch of stuff we don't need. Depending on your application,
+      # you may have to drop some of these (e.g. *.pod).
+    cleanup:
+        - '/bin/corelist'
+        - '/bin/cpan'
+        - '/bin/enc2xs'
+        - '/bin/encguess'
+        - '/bin/h2ph'
+        - '/bin/h2xs'
+        - '/bin/instmodsh'
+        - '/bin/json_pp'
+        - '/bin/libnetcfg'
+        - '/bin/perlbug'
+        - '/bin/perldoc'
+        - '/bin/perlivp'
+        - '/bin/perlthanks'
+        - '/bin/piconv'
+        - '/bin/pl2pm'
+        - '/bin/pod*'
+        - '/bin/prove'
+        - '/bin/ptar*'
+        - '/bin/shasum'
+        - '/bin/splain'
+        - '/bin/xsubpp'
+        - '/bin/zipdetails'
+        - '/include'
+        - '/man'
+        - '*.pod'
+
+  # Installs the modules generated by flatpak-cpan-generator.
+  # Note that *any* Makefile.PL-style modules MUST be installed in this one step,
+  # as once perllocal.pod is written in one module, it cannot be modified by others.
+  - name: perl-libs
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    # Same as with the Perl module, we need to restore write permission.
+    # However, -f is now passed to avoid errors from trying to touch files from the
+    # above module that are now marked as read-only.
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+    sources:
+      - generated-sources.json
+    # This step should be customized based on the CPAN packages you're using.
+    cleanup:
+      - '/bin'
+      - '/man'
+  - name: "Hiperthermia"
+    buildsystem: simple
+    build-commands:
+      - 'cd HiperthermiaSource; perl lib/Exd/Fontconfig.pm'
+      - 'cd HiperthermiaSource; perl lib/Exd/FileFormat/Fontconfig.pm'
+      - 'cd HiperthermiaSource; perl -Ilib lib/Exd/Gui.pm'
```

Comment:
This lines `cd HiperthermiaSource; perl -Ilib lib/Exd/Gui.pm` exist because my program uses Inline C and a C compiler is not available in runtime so modules need to be run at least once during flatpak setup so the embeded C gets compiled.